src/components/ClusterTemplateQuotas/NewQuotaDialog/AccessFields.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import {
+  clusterRoleGroupVersionKind,
+  clusterTemplatesRoleRef,
+  groupGVK,
+  userGVK,
+} from '../../../constants';
+import InlineResourceLink from '../../../helpers/InlineResourceLink';
+import MultiSelectField from '../../../helpers/MultiSelectField';
+import PopoverHelpIcon from '../../../helpers/PopoverHelpIcon';
+import { SelectInputOption } from '../../../helpers/SelectField';
+import { useTranslation } from '../../../hooks/useTranslation';
+
+import { sortByResourceName } from '../../../utils/utils';
+import { K8sResourceCommon, useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+import { Group, User } from '../../../types';
+import { useAddAlertOnError } from '../../../alerts/useAddAlertOnError';
+import { Trans } from 'react-i18next';
+
+export const useUsers = () => {
+  return useK8sWatchResource<User[]>({
+    groupVersionKind: userGVK,
+    isList: true,
+  });
+};
+
+export const useGroups = () =>
+  useK8sWatchResource<Group[]>({
+    groupVersionKind: groupGVK,
+    isList: true,
+  });
+
+const getOptions = (resources: K8sResourceCommon[]): SelectInputOption[] =>
+  sortByResourceName(resources).map((resource) => ({
+    value: resource.metadata?.name,
+    disabled: false,
+  }));
+
+const UsersHelpText = () => {
+  return (
+    <Trans ns="plugin__clustertemplates-plugin">
+      Create a RoleBinding granting the selected users the
+      <InlineResourceLink
+        groupVersionKind={clusterRoleGroupVersionKind}
+        name={clusterTemplatesRoleRef.name}
+      />
+      &nbsp; role in the selected namespace. This role provides the minimum permissions needed for
+      using the cluster templates.
+    </Trans>
+  );
+};
+
+export const UsersField = () => {
+  const { t } = useTranslation();
+  const [users, loaded, error] = useUsers();
+  useAddAlertOnError(error, t('Failed to load users'));
```

Comment:
when language is changed, `t` will be updated and it will result in adding a new alert. We should store only `keys` and get the actual translation in component.
```suggestion
  // t('Failed to load users')
  useAddAlertOnError(error, 'Failed to load users');
```

Reply:
You missed only this line to remove the `t` :)